PWG/EMCAL/EMCALtasks/AliEmcalCorrectionCellTimeCalib.cxx

```diff
@@ -56,6 +56,10 @@ Bool_t AliEmcalCorrectionCellTimeCalib::Initialize()
   if (!fRecoUtils)
     fRecoUtils  = new AliEMCALRecoUtils;
     
+
+  if (fDoMergedBCs)
```

Comment:
The value should be initialized from YAML in this function before here. See the clusterizer component to see how and where to do this.

Remember to define the value in the default YAML file